scripts/migrate-test.sh

```diff
@@ -4,5 +4,19 @@ set -e
 HERE=$(dirname $0)
 . $HERE/migrate-common.sh
 
+
+rm ${PWD}/demo -rf
+rm ${PWD}/git -rf
+
+ORDERLY_IMAGE=vimc/orderly:master
+docker pull $ORDERLY_IMAGE
+docker run --rm \
+  --entrypoint create_orderly_demo.sh \
```

Comment:
migrate-test.sh was failing because it didn't have the whole orderly db to work with. The line which copied existing published state from report-version failed because report_version table didn't exist. So creating the db here, as it already is in run-dependencies etc.